spec/requests/v1/recipes_spec.rb

```diff
@@ -89,5 +127,60 @@
         run_test!
       end
     end
+
+    put 'Edits a recipe' do
+      tags 'Recipes'
```

Comment:
Why did you choose to only test this with swagger?

Reply:
I'm waiting for the other PR to be merged